data/hails.txt

```diff
@@ -1824,60 +1841,14 @@ phrase "friendly pirate"
 		" I wish I could afford more"
 		" I'm going to need to make money to buy more"
 		" Best experience I've ever had"
+		" It's unreal."
+		" It'll make you forget you're alive."
+		" It'll make you soar."
```

Comment:
these 3 lines (1844-1846) have an interpunction after them, while the next (final) word already gives it an interpunction, so remove these

Reply:
I'll note that the empty word at the top results in something like `substance?.` or `drug?!`.